gadgets/advise_networkpolicy/gadget.yaml

```diff
@@ -10,3 +10,5 @@ datasources:
       ebpf.map.flush-on-stop: true
       generate_networkpolicy.enable: true
       kubenameresolver.enable: true
+paramDefaults:
```

Comment:
For eBPF params, how does it interact with the `defaultValue` in params section?

```yaml
params:
  ebpf:
    ppid:
      key: ppid
      defaultValue: "123"
      description: Parent process ID
paramDefaults:
  operator.oci.ebpf.ppid: "999"
```

Maybe remove `params.<operator>.defaultValue` and fully rely on this new `paramDefaults`.

BTW, I created few quick unit-tests to validate the behaviour, but I don't know. Using a gadget seems too much, but I didn't find a simpler way to cover it all: https://github.com/inspektor-gadget/inspektor-gadget/commit/5aa0fa334ce15885bc3e8ce7cb0b2ecb91f8ca9f